Add missing documentation for linting rules (#4001)
Azure/typespec-azure: website/src/content/docs/docs/libraries/azure-resource-manager/rules/arm-resource-operation.md

---
title: "arm-resource-operation"
---

```text title="Full name"
@azure-tools/typespec-azure-resource-manager/arm-resource-operation
```

> Validate ARM Resource operations. All resource operations must be defined inside an interface, include an `api-version` parameter, and use the correct decorator for the HTTP verb.

Reviewer: ```suggestion
Validate that all ARM Resource operations are defined inside an interface, include an `api-version` parameter, and use the correct decorator for the HTTP verb.
```
